### DISCUSSION TITLE
Very new to python and this is the one time im gonna be using it

### DISCUSSION THREAD

**Discussion by splipopgus** [openai/whisper#897] [Q&A] [answered]
Tried using the command pip install git+https://github.com/openai/whisper.git, and it ran it pretty smoothly at first and then I got 3 errors

ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11
ERROR: Could not find a version that satisfies the requirement torch (from openai-whisper) (from versions: none)
ERROR: No matching distribution found for torch

I have downloaded pip, pytorch, and python so everything ought to be up to date, from my limited knowledge of coding I guess I need to get Whisper API key or something? Idk what should I do fellas

**Comment by jongwook** [comment, marked as answer]
You might have installed Python 3.11 which is not supported by many libraries Whisper uses. While they'll be gradually adding support for 3.11, you'll probably have better results with Python 3.9 or 3.10.

I made an edit in README.md to clarify that only Python 3.8-3.10 is expected to work (5c1a8c10e762bf9c29fcf6b3e40f17bc8ab09864).
